pkg/engine/templates_generated.go

```diff
@@ -16525,7 +16525,8 @@ var _k8sKubernetesparamsT = []byte(`    "etcdServerCertificate": {
          "19.03.12",
          "19.03.13",
          "19.03.14",
-         "20.10.5"
+         "20.10.5",
```

Comment:
Should this just be a value that is a default and then, if the user picks something else, the code just uses it?  As it is, I doubt that we have done regression testing against all of these versions and the current kuberenetes/aks-engine versions.